sdk/eventgrid/eventgrid/.vscode/settings.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I don't think many clients have a VSCode `settings.json` - were you not getting proper behavior without this?

Reply:
I was not, but it's possible that I am doing something wrong (I am unfortunately still a neophyte when it comes to a bunch of the tooling in the JavaScript world).

While I did add this entire library to the `dataplane.code-workspace` file in the root, for my inner loop, I navigate to the `sdk/eventgrid/eventgrid` folder, and just do `code .` to launch an instance of VS Code instead of opening the workspace with all the data plane packages.

Since I have not personally configured any user specific settings, I wasn't getting the nice behavior without this (auto format wouldn't happen, etc).

If the recommendation is to just set these all as user settings, I'm happy to do that (I don't work on other projects that don't follow our guidelines).

Note that I launch the editor from inside the `sdk/eventgrid/eventgrid` folder because VS Code feels a little sluggish when I open the entire code workspace, and I usually don't need to look at other projects when I am doing my inner loop work.

So, just let me know what's preferable, and I'll do it.

Reply:
The hope was that the workspace would be small enough (track2 only) that it wouldn't be slow. Sorry to hear that it still is.